llvm/test/CodeGen/AArch64/nofpclass.ll

```diff
@@ -21,9 +21,28 @@ entry:
   ret <4 x float> %c
 }
 
-define { float, float } @struct({ float, float } nofpclass(nan) %a) {
+define { float, float } @struct({ float, float} nofpclass(nan) %a) {
 ; CHECK-LABEL: struct:
 ; CHECK:       // %bb.0:
 ; CHECK-NEXT:    ret
    ret {float, float} %a
 }
+
+%struct.f2 = type { float, float }
```

Comment:
Test the array case, but what I was asking was to test this struct type as the input